service-commands/scripts/userClockValues.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Find the creator nodes associated with a user and their respective clock values.
+ *
+ * Script usage: node userClockValues.js <size> <outFile>
+ */
+const axios = require('axios')
+const CreatorNode = require('@audius/libs/src/services/creatorNode')
+const { Command } = require('commander')
+
+function commaSeparatedList(value, dummyPrevious) {
```

Comment:
can we update `dummyPrevious` with a more descriptive name?

Reply:
Oops, sorry, this was copied code from the example. Updating 👍